### ISSUE TITLE
[How To] Populate DB with test/dummy data

### ISSUE THREAD

**Issue by punndcoder28** [esnvutbrno/buena-fiesta#166]
I am building the project locally to test out the project and wanted to know how I can populate the database with dummy values. The place I am facing issue while completing my profile locally and the `Universities` and `Faculty` cannot be selected since there are no options to select from. Hence, I would like to know how I can pre-fill the database with test data. I have followed the instructions on the `README.md` file before starting the development server.

**Comment by thejoeejoee** [comment]
Fiesta does have set of [factory boy](https://factoryboy.readthedocs.io/en/latest/index.html) factories to generate data: [fiesta/apps/utils/factories](https://github.com/esnvutbrno/buena-fiesta/tree/develop/fiesta/apps/utils/factories) -- especially the `UniversityFactory` and `FacultyFactory` are useful for you, but there are also others for different models.

And for the generating, I usually edited the [`seed` command ](https://github.com/esnvutbrno/buena-fiesta/blob/22b328f7b0b4e5df54a07854c5296d8691f3694d/fiesta/apps/utils/management/commands/seed.py#L10-L10) to generate models for my demand -- feel free to customize it and generate whatever you want. By `make seed` you can call that command.

**Comment by punndcoder28** [comment]
Thanks for pointing me in the right direction! I was able to generate mock data for universities, but the `FacultyFactory` seems to be missing in the codebase. Could have not been pushed to the `develop` branch. Will check the DB schema, other factory files and try to write the `FacultyFactory` class which can be used to generate mock faculty information

**Comment by thejoeejoee** [comment]
@punndcoder28 I pushed it just a moment before I replied you: https://github.com/esnvutbrno/buena-fiesta/commit/22b328f7b0b4e5df54a07854c5296d8691f3694d 

Just update your fork and it should be there.

**Comment by punndcoder28** [comment]
Thanks for the quick response. Can I submit a PR moving the `FacultyFactory` into another file from the current `UniversityFactory` so for better readability?

**Comment by punndcoder28** [comment]
Open PR #170 which refactors `FacultyFactory` into a standalone file

**Comment by punndcoder28** [comment]
Closing this in favor of #170
